src/App.js

```diff
@@ -12,8 +13,8 @@ function App() {
       <Container>
         <Router>
           <Route path="/" component={Navbar} />
-          {/* <Route exact path="/" component={Home} />
-          <Route path="/requests" component={Requests} />
+          <Route exact path="/" component={Hero} />
```

Comment:
hmm I think the Hero component just needs to be rendered inside the Home container instead of replacing it in the main route